vignettes/data-combined.Rmd

```diff
@@ -0,0 +1,300 @@
+can define the name when adding the data, as in the above example adding simulated 
+results.
+
+There are a few things to keep in mind here:
+
+- It doesn't matter in which order observed or simulated datasets are added.
+- You can add multiple `DataSet` objects in `$addDataSets()` method call.
+- You can add only a single instance of `SimulationResults` in `$addSimulationResults()` method call.
+- If you add a dataset with the same name as existing (e.g., adding simulation results 
+from multiple simulations and not specifying the name, so the path of the results 
+is used as the name), the new data will replace the existing.
+
+## Grouping
+
+Since the `DataCombined` object can store many datasets, some of them may naturally form a grouping and you would wish to track this in the object. Both `$addDataSets()` and `$addSimulationResults()` allow group specification via `groups` argument.
+
+When being plotted, data sets without a grouping will appear with distinct color, 
+line style (for simulated results), or symbol style (for observed data) with a 
+separate entry in the legend, with the name of the data set being the legend entry.
+
+Let's create a `DataCombined` and add simulation results and observed data *without* 
+specifying their grouping and create a time profile:
+
+```{r}
+myDataCombined <- DataCombined$new()
+
+myDataCombined$addSimulationResults(
+  simulationResults = simResults,
+  quantitiesOrPaths = outputPath,
+  names = "Aciclovir Plasma"
+)
+
+myDataCombined$addDataSets(
+  obsData$`Vergin 1995.Iv`
+)
+
+plotIndividualTimeProfile(dataCombined = myDataCombined)
+```
+
+If you do not specify `groups` when you add datasets or set it to `NULL`, and 
+wish to update groupings later, you can use the `$setGroups()` method. All data 
+within one group will get one entry in the legend with the name of the group, 
+and will be plotted with the same color.
+
+```{r}
+myDataCombined$setGroups(names = c("Aciclovir Plasma", obsData$`Vergin 1995.Iv`$name),
+                         groups = "Aciclovir PVB")
+
+plotIndividualTimeProfile(dataCombined = myDataCombined)
+```
+
+At any point, you can check the current names and groupings with the following active field:
+
+```{r}
+myDataCombined$groupMap
+```
+
+## Transformations
+
+Sometimes the raw data included in `DataCombined` needs to be transformed using specified
+offset and scale factor values.
+
+This is supported via `$setDataTransformations()` method, where you can specify the names of datasets, and offsets and scale factors. If these arguments are scalar (i.e., of length 1), then the same value will be applied to all specified names.
+
+The internal data frame in `DataCombined` will be transformed with the specified parameters, with the new columns computed as: 
+
+  - For x values: `newXValue = (rawXValue + xOffset) * xScaleFactor`
+  - For y values: `newYValue = (rawYValue + yOffset) * yScaleFactor`
+  - For arithmetic error: `newErrorValue = rawErrorValue * scaleFactor`
```

Comment:
should be `... * xScaleFactor`

Reply:
Updated.

Reply:
**abs**(yScaleFactor) 😉 (was fixed in the code but not in the docu)